content/cpp/concepts/operator-overloading/operator-overloading.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: 'Operator Overloading'
+Description: 'Operator overloading redefines operators for user-defined classes.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Operators'
+  - 'Binary'
+  - 'Logical'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+**Operator overloading** redefines operators for user-defined classes. When an overloaded operator is called, the compiler determines which definition to use based on the arguments provided. The following operators can be overloaded:
+
+| ->      | ->\* | new    | new [] | delete | delete []     |
```

Comment:
The table is sort of cryptic. Can we have some description on what the layout, columns and rows, actually means?

Reply:
@SSwiniarski These are all the operators that could be overloaded. I was going to list them but thought it would be too long so that's why I put it in a table. What do you suggest?

Reply:
@KTom101 A bulleted list would be better.